src/xcoobee/api/ApiAccessTokenCache.js

```diff
@@ -44,22 +43,22 @@ class ApiAccessTokenCache {
    */
   get(apiUrlRoot, apiKey, apiSecret, fresh) {
     if (!apiUrlRoot) {
-      throw TypeError('apiUrlRoot is required.');
+      return Promise.reject(new TypeError('apiUrlRoot is required.'));

```

Comment:
Since this is returning a rejected Promise, would you please update the JSDoc to not say it throws a TypeError.